### PR TITLE
fix biceps.c11, biceps.c12, biceps.c13, biceps.c14 and biceps.c15

### PR DESCRIPTION
**Pull request by belagertem**
the requirement tests biceps.c11 - biceps.c15 no longer fail when a description modification report with the same state and mdib version is seen. Unit tests added for that scenario.

# Checklist

The following aspects have been respected by the author of this pull request, confirmed by both pull request assignee **and** reviewer:

* Adherence to coding conventions
  * [x] Pull Request Assignee
  * [x] Reviewer
* Adherence to javadoc conventions
  * [x] Pull Request Assignee
  * [x] Reviewer
* Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* README update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* config update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* SDCcc executable ran against a test device (if necessary)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java [hunk 1]
```diff
@@ -694,59 +698,12 @@ void testRequirementR5053() throws NoTestData, IOException {
             + " EpisodicAlertReport.")
     @RequirePrecondition(simplePreconditions = {ConditionalPreconditions.TriggerEpisodicAlertReportPrecondition.class})
     void testRequirementC11() throws NoTestData, IOException {
-        final var mdibHistorian = mdibHistorianFactory.createMdibHistorian(
-                messageStorage, getInjector().getInstance(TestRunObserver.class));
-
-        final var acceptableSequenceSeen = new AtomicInteger(0);
+        final GetStatesOfReportParts getStatesOfReportParts = report -> ((AbstractAlertReport) report)
```
[the remark this answers is not in the document]
Comment (belagertem) [reply sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java]: true, changed it to `EpisodicAlertReport`

File: sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java [hunk 2]
```diff
@@ -154,8 +157,7 @@ void testRequirementR00550() throws NoTestData {
     @TestIdentifier(EnabledTestConfig.BICEPS_C5)
     @TestDescription("Starting from the initially retrieved mdib, applies each episodic report to the mdib and checks"
             + " for each AbstractDescriptor contained in a DescriptionModificationReport"
-            + " that it was inserted or deleted or updated by changing"
-            + " at least one child or attribute.")
+            + " that it was inserted or deleted or updated by changing" + " at least one child or attribute.")
```
Comment (ldeichmann) [inline sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java:160]: That is really odd

File: sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java [hunk 3]
```diff
@@ -957,61 +759,132 @@ void testRequirementC14() throws NoTestData, IOException {
...
-                                            stateAfterReport.orElseThrow(),
-                                            stateBeforeReport.orElseThrow(),
-                                            String.format(STATE_UNCHANGED, operationState.getDescriptorHandle()));
-                                }
+                    final var reportParts = getStatesOfReportParts.apply(reportClass.cast(report));
+                    for (var reportPart : reportParts) {
+                        for (var state : reportPart) {
+                            final Optional<? extends AbstractState> stateBeforeReport;
+
+                            stateBeforeReport = mdibAccess.getState(state.getDescriptorHandle(), stateClass);
```
Comment (ldeichmann) [inline sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java:868]: The previous version that made the distinction between `AbstractMultiState` and `AbstractState` was better, how can you retrieve the correct multi state handle now? This only queries by descriptor handle.

File: sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java (hunk 2 above)
Comment (belagertem) [reply sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java]: don't know why this happend, fixed

File: sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java (hunk 3 above)
Comment (belagertem) [reply sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java]: True, changed it back

**Review by ldeichmann — APPROVED** [review]

**Pull request merged**